components/playground/instance/tiflash.go

```diff
@@ -131,26 +132,37 @@ func (inst *TiFlashInstance) Start(ctx context.Context, version utils.Version) e
 
 	args := []string{
 		"server",
-	}
-	args = append(args,
 		fmt.Sprintf("--config-file=%s", configPath),
 		"--",
-		fmt.Sprintf("--tmp_path=%s", filepath.Join(inst.Dir, "tmp")),
-		fmt.Sprintf("--path=%s", filepath.Join(inst.Dir, "data")),
-		fmt.Sprintf("--listen_host=%s", inst.Host),
-		fmt.Sprintf("--logger.log=%s", inst.LogFile()),
-		fmt.Sprintf("--logger.errorlog=%s", filepath.Join(inst.Dir, "tiflash_error.log")),
-		fmt.Sprintf("--status.metrics_port=%d", inst.StatusPort),
-		fmt.Sprintf("--flash.service_addr=%s", utils.JoinHostPort(AdvertiseHost(inst.Host), inst.ServicePort)),
-		fmt.Sprintf("--raft.pd_addr=%s", strings.Join(endpoints, ",")),
-		fmt.Sprintf("--flash.proxy.addr=%s", utils.JoinHostPort(inst.Host, inst.ProxyPort)),
-		fmt.Sprintf("--flash.proxy.advertise-addr=%s", utils.JoinHostPort(AdvertiseHost(inst.Host), inst.ProxyPort)),
-		fmt.Sprintf("--flash.proxy.status-addr=%s", utils.JoinHostPort(inst.Host, inst.ProxyStatusPort)),
-		fmt.Sprintf("--flash.proxy.data-dir=%s", filepath.Join(inst.Dir, "proxy_data")),
-		fmt.Sprintf("--flash.proxy.log-file=%s", filepath.Join(inst.Dir, "tiflash_tikv.log")),
-	)
-
-	var err error
+	}
+	runtimeConfig := [][]string{
+		{"tmp_path", filepath.Join(inst.Dir, "tmp")},
```

Comment:
I think tmp_path can be removed, TiFlash's default behavior "${path}/tmp" can handle it.

Reply:
removed https://github.com/pingcap/tiup/pull/2346/commits/8ec2aa8e13a072c5174920607b09e7c5be1b790a